rest/model/host.go

```diff
@@ -67,6 +67,7 @@ type DistroInfo struct {
 	IsVirtualWorkstation bool    `json:"is_virtual_workstation"`
 	User                 *string `json:"user"`
 	IsWindows            bool    `json:"is_windows"`
+	BootstrapMethod      *string `json:"method"`
```

Comment:
Can you spell this out as "bootstrap_method" in the json?

Reply:
The rest of the fields are keeping it consistent with the Json in the Struct it's from. Is it a good idea to change it here?